src/__tests__/mailtrap_client.test.ts

```diff
@@ -111,4 +114,45 @@ describe("MailtrapClient#send", () => {
       expect(result).toEqual(successData);
     });
   });
+
+  describe("server error", () => {
+    it("throws error returned from the server", async () => {
+      mock.onPost("https://send.api.mailtrap.io/api/send").reply(400, {
+        success: false,
+        errors: ["subject is missing", "message is missing"],
+      });
+
+      const client = new MailtrapClient({ token: "MY_API_TOKEN" });
+
+      try {
+        await client.send(goodMail);
+      } catch (err) {
+        expect(err).toBeInstanceOf(MailtrapError);
+        if (err instanceof MailtrapError) {
+          expect(err.message).toEqual("subject is missing, message is missing");
+          // @ts-expect-error ES5 types don't know about cause property
```

Comment:
```suggestion
          // @ts-expect-error ES5 types doesn't know about cause property
```
Same below

Reply:
um, types don't know... if it would be _type_ the it _doesn't_... grammatically it's correct, could it be "type"? sure